From 90845a3a263e0f37b9c756c0b01377b9d1a225c9 Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd.kuhls@t-online.de>
Date: Sun, 31 Jul 2016 10:37:05 +0200
Subject: [PATCH] vpx_mem/vpx_mem.h: do not include <lddk.h>

The <lddk.h> header does not exist in most C libraries, and including
it causes build failures.

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 vpx_mem/vpx_mem.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/vpx_mem/vpx_mem.h b/vpx_mem/vpx_mem.h
index c14f288..82df745 100644
--- a/vpx_mem/vpx_mem.h
+++ b/vpx_mem/vpx_mem.h
@@ -12,9 +12,6 @@
 #define VPX_VPX_MEM_VPX_MEM_H_
 
 #include "vpx_config.h"
-#if defined(__uClinux__)
-#include <lddk.h>
-#endif
 
 #include <stdlib.h>
 #include <stddef.h>
-- 
2.8.1

